.github/DISCUSSION_TEMPLATE/q-a.yml

```diff
@@ -1,4 +1,3 @@
-title: ""
```

Comment:
Wouldn't you need to move the dash down to be in front of "labels", so that the doc starts with a list item, still? Without it, it becomes an object/dictionary/map.

Reply:
I also was confused by that one, but there is/was no dash at all: https://github.com/PrivateBin/PrivateBin/blob/master/.github/DISCUSSION_TEMPLATE/q-a.yml
This is just GitHub showing it as a diff for removed line. The next line also has no indentation, so no that does not work.

And yeah I sharp again had to cross-check.

Reply:
Doh! I apologize for the noise.